src/targets/node/nodeTarget.ts

```diff
@@ -57,7 +58,8 @@ export class NodeTarget implements Target {
     if (targetInfo.title)
       this._targetName = `${basename(targetInfo.title)} [${targetInfo.targetId}]`;
     else this._targetName = `[${targetInfo.targetId}]`;
-
+    if (skipFiles.length)
+      this._scriptSkipper = new ScriptSkipper(skipFiles);
```

Comment:
If each Target has a ScriptSkipper, does that mean that if I toggle a script's skip state, that only applies to one target? It seems like that should apply to the whole debug session. But, I'm not totally sure about that...